doozer/doozerlib/backend/konflux_fbc.py

```
@@ -1147,6 +1151,64 @@ def _extract_git_commits_from_nvrs(self, bundle_nvrs: str) -> list[str]:
+
+        :param pullspec: Image pullspec to check
+        :param logger: Logger instance
+        :return: True if image exists, False otherwise
+        """
+        logger = logger or self._logger
+        try:
+            logger.debug(f"Checking if image exists: {pullspec}")
+            registry_config = os.getenv("KONFLUX_ART_IMAGES_AUTH_FILE")
+            await util.oc_image_info_for_arch_async(pullspec, registry_config=registry_config)
+            logger.debug(f"Image exists: {pullspec}")
+            return True
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)